src/OnboardingSPA/data/routes/default-flow.js

```diff
@@ -327,6 +327,8 @@ export const steps = [
 		VIEW: VIEW_DESIGN_THEME_STYLES_PREVIEW,
 		designDrawerActiveLinkIncludes: '/wp-setup/step/design/theme-styles/',
 		patternId: 'theme-styles',
+		excludeNextStepsFromRouter: true,
+		numberOfStepsToSkip: 2,
```

Comment:
I don't think using `numberOfStepsToSkip` is the right way to do this, let's discuss this once.